### PR TITLE
Add Disable Rebuild in Rebuild Settings

### PR DESCRIPTION
**Pull request by hagzag**
There is a use case where you want to disable the rebuild for a certain project without removing the rebuild plugin.

### CODE REVIEW

**Comment by hagzag** [comment]
@rsandell Any update on this ?

File: src/main/java/com/sonyericsson/rebuild/RebuildAction.java [hunk 1]
```diff
@@ -361,11 +361,21 @@ public boolean isRebuildAvailable() {
                 && project.hasPermission(Item.BUILD)
                 && project.isBuildable()
                 && project instanceof Queue.Task
-                && !isMatrixRun();
+                && !isMatrixRun() 
+                && isAutoRebuildDisbaled();
```
Comment (rsandell) [inline src/main/java/com/sonyericsson/rebuild/RebuildAction.java]: the name suggests that you should return ```!isAutoRebuildDisbaled()```

File: src/main/java/com/sonyericsson/rebuild/RebuildSettings.java [hunk 2]
```diff
@@ -41,14 +41,16 @@
      * boolean auto rebuild variable.
      */
     private boolean autoRebuild;
+    private boolean autoRebuildDisabled;
```
Comment (rsandell) [inline src/main/java/com/sonyericsson/rebuild/RebuildSettings.java]: I think this is bad naming and leads at me a bit confused. It suggests that the setting disables the ```autoRebuild``` but the description says something else.

File: src/main/java/com/sonyericsson/rebuild/RebuildAction.java [hunk 3]
```diff
@@ -361,11 +361,21 @@ public boolean isRebuildAvailable() {
...
 
     }
 
-    /**
+    private boolean isAutoRebuildDisbaled() {
+        RebuildSettings settings = (RebuildSettings)getProject().getProperty(RebuildSettings.class);
+        
+        if (settings != null && settings.getAutoRebuildDisabled()) {
+			return false;
```
Comment (rsandell) [inline src/main/java/com/sonyericsson/rebuild/RebuildAction.java]: inverse logic?

**Pull request merged**